lte/gateway/deploy/roles/agw_docker/tasks/main.yml

```diff
@@ -0,0 +1,195 @@
+      {{ magma_pkgrepo_dist }} {{ magma_pkgrepo_component }}
+    dest: /etc/apt/sources.list.d/{{ magma_pkgrepo_name }}.list
+  become: yes
+
+- name: Add unvalidated Apt signing key.
+  when: magma_pkgrepo_key_fingerprint == ""
+  become: yes
+  block:
+    - name: Download key
+      apt_key:
+        url: "{{ magma_pkgrepo_key }}"
+        state: present
+        validate_certs: no
+    - name: Ignore server ssl cert
+      copy:
+        dest: /etc/apt/apt.conf.d/99insecurehttpsrepo
+        content: |
+          Acquire::https::{{ magma_pkgrepo_host }} {
+          Verify-Peer "false";
+          Verify-Host "false";
+          };
+
+- name: Add validated Apt signing key.
+  when: magma_pkgrepo_key_fingerprint != ""
+  become: yes
+  apt_key:
+    url: "{{ magma_pkgrepo_key }}"
+    id: "{{ magma_pkgrepo_key_fingerprint }}"
+    state: present
+
+- name: Update and upgrade apt packages
+  become: yes
+  apt:
+    update_cache: yes
+
+- name: Install runtime dependencies.
+  become: yes
+  apt:
+    name: "{{ packages }}"
+  vars:
+    packages:
+      - graphviz
+      - python-all
+      - module-assistant
+      - openssl
+      - dkms
+      - uuid-runtime
+
+- name: Preconfigure wireshark (tshark) SUID property
+  become: yes
+  ignore_errors: yes
+  shell: bash -c 'echo "wireshark-common wireshark-common/install-setuid boolean true" | debconf-set-selections'
+
+- name: Install prebuilt openvswitch packages
+  become: yes
+  apt:
+    name: "{{ packages }}"
+    #dpkg_options: 'force-confold,force-confdef,force-overwrite'
+  vars:
+    packages:
+      - libopenvswitch
+      - openvswitch-datapath-dkms
+      - openvswitch-datapath-source
+      - openvswitch-common
+      - openvswitch-switch
+
+- name: Install Docker
+  apt:
+    name: "{{ packages }}"
+    state: present
+    update_cache: yes
+  vars:
+    packages:
+    - docker-ce
+    - docker-ce-cli
+    - containerd.io
+
+- name: Download docker-compose {{ docker_compose_version }}
+  get_url:
+    url : https://github.com/docker/compose/releases/download/{{ docker_compose_version }}/docker-compose-Linux-x86_64
+    dest: ~/docker-compose
+    mode: '+x'
+
+- name: Check docker-compose exists
+  stat: path=~/docker-compose
+  register: docker_compose
+
+- name: Move docker-compose to /usr/local/bin/docker-compose
+  command: mv ~/docker-compose /usr/local/bin/docker-compose
+  when: docker_compose.stat.exists
+
+- name: create directories for magma configs and files
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: '0755'
+  loop:
+    - "/var/opt/magma"
+    - "/var/opt/magma/configs"
+    - "/var/opt/magma/certs"
+    - "/etc/magma"
+    - "/etc/magma/templates/"
+    - "/var/opt/magma/docker"
+
+- name: Enable IP forwarding
+  become: true
+  sysctl: name="net.ipv4.ip_forward" value=1 sysctl_set=yes state=present reload=yes
+
+- name: touch snowflake
+  shell: touch /etc/snowflake
```

Comment:
SNOWFLAKE_PATH perhaps? It appears to be referenced in the docker-compose file as well.

Reply:
there will be no customisation required for snowflake path.
removed SNOWFLAKE_PATH.